CControl_Handler.cc

```diff
@@ -57,29 +57,63 @@ int CControl_Handler::DeviceArm(int run, Options *opts){
     fStatus = DAXHelpers::Idle;
     return -1;
   }else{
+     fBoardHandle = fV2718->GetHandle();
      std::cout << "V2718 Initialised" << std::endl;
   }
 
 // ----------------------------------------------
   // Getting options for DDC10 HEV module
   HEVOptions hopts;
+  
+  std::vector<BoardType> dv = fOptions->GetBoards("DDC10", fProcname);
+  // Init DDC10 only when included in config - only for TPC   
+  if (dv.size() == 1){
+     if(fOptions->GetHEVOpt(hopts) == 0){
+        fDDC10 = new DDC10();
+	if(fDDC10->Initialize(hopts) != 0){
+           fLog->Entry(MongoLog::Error, "Failed to initialise DDC10 HEV");
+	   fStatus = DAXHelpers::Idle;
+	   return -1;
+	}else{
+	   std::cout << "DDC10 Initialised" << std::endl;
+	}
+     }else{
+	fLog->Entry(MongoLog::Error, "Failed to pull DDC10 options from file");
+     }
+  }
 
-  if(fOptions->GetHEVOpt(hopts) == 0){
-
-    fDDC10 = new DDC10();
-    if(fDDC10->Initialize(hopts)!=0){
-      fLog->Entry(MongoLog::Error, "Failed to initialise DDC10 HEV");
-      fStatus = DAXHelpers::Idle;
-      return -1;
-    }else{
-      std::cout << "DDC10 Initialised" << std::endl;
-    }
+ 
+  // Getting options for the Muon Veto V1495 board
+  // Init V1495_MV only when included in config - Muon Veto only
+  std::vector<BoardType> mv = fOptions->GetBoards("V1495", fProcname);
+  BoardType mv_def = mv[0];
+  fBID = mv_def.board;
+  if (mv.size() == 1){
+     	fV1495 = new V1495(fLog, fOptions, mv_def.board, fBoardHandle, mv_def.vme_address);
+	// Writing registers to the V1495 board
+	for(auto regi : fOptions->GetRegisters(fBID)){
```

Comment:
What do we do if a V1495 has the same SN as a V1724? Might future-proof us to add a GetV1495Opts routine.